vllm_ascend/attention.py

```diff
@@ -577,13 +577,11 @@ def forward(
                                                self.num_kv_heads,
                                                self.head_size)
                 slots = attn_metadata.slot_mapping
-                torch_npu.npu_reshapecache(key=key,
-                                           value=value,
-                                           keyCache=key_cache,
-                                           valueCache=value_cache,
-                                           slotMapping=slots,
-                                           compressType=0,
-                                           kvCacheCfg=0)
+                torch_npu._npu_reshape_and_cache(key=key,
```

Comment:
so this change rely on a new torch-npu version, right?

And why the interface is private？